org.tribler.Tribler.yaml

```diff
@@ -19,7 +19,9 @@ finish-args:
   - --filesystem=xdg-download                           # Store downloads in the user's download folder
   - --persist=.Tribler                                  # Store the application state
   - --talk-name=org.kde.StatusNotifierWatcher           # Used for AppIndicator support
+  - --env=SSL_CERT_DIR=/etc/ssl/certs
```

Comment:
Are there settings when this is actually another directory? Is `/etc/ssl/certs` stable through different OSs?

Reply:
It is not stable, at all. However, this will at least get Ubuntu working (and any others that use the same certificate directory).

Open to suggestions, if you know of a way to also get other distros working.